libtransact/src/context/manager.rs

```diff
@@ -173,8 +162,8 @@ impl<
     pub fn set_state(
         &mut self,
         context_id: &ContextId,
-        key: K,
-        value: V,
+        key: &str,
```

Comment:
These should be owned `String` and `Vec<u8>`, as the previous types were owned.

Reply:
Addressed